spec/export/index.md

```diff
@@ -210,6 +210,7 @@ Retry-After: 120</code></pre></td>
       <td>Returned by the server if the export operation fails.</td>
       <td><pre><code>Status: 500 Internal Server Error
 Content-Type: application/json
+Retry-After: 120
```

Comment:
The failed example shouldn't have a retry-after header -- once failed, the request is done.

Reply:
fixed https://github.com/HL7/bulk-data/pull/32/commits/e39e47500036c21b01b50a4033cf054ee69f7437